xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -266,6 +278,20 @@ def _skip_target(
         if attr in deps_attrs and info.xcode_target
     ]
 
+    def _target_replacement_label(info):
+        if not info.xcode_target:
+            return target.label
+        if _APPLE_INTERNAL_TEST_BUNDLE_SUFFIX not in info.xcode_target.label.name:
```

Comment:
We also discussed log a warning if the suffix is not present, so I'll probably move the skip type check up and add the warning here

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2196/commits/2044fa8c9e816ed30e0f0aed1637b2a1755079b9